src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -245,10 +222,10 @@ const getAvailablePaymentMethods = (
           </>
         }
       >
-        <USBankOnlyLabel ml={0.5} variant="xs">
+        <Text ml="24px" variant="xs" color="black60">
```

Comment:
@AlicanAkyuz how would you feel about including this in the `label` (I think we could basically wrap this + the other text in a `Flex`) to avoid having to use a custom `px` margin value that's not in our design system?

Reply:
@mdole An issue with this is the label has a hover underline that we don't want for description.

Reply:
Ideally, we would create the option to add an icon to radio labels, but this hasn't been prioritized.

Reply:
@mdole yeah that's a good suggestion! we actually thought about this as Rachel pointed out, but it got de-prioritised (it took a lot of back-and-forth discussions to come up with that label in the first place...).

Reply:
I'm going to merge this as-is, let's create a ticket on our backlog to do the palette work (if there isn't already one).

Reply:
gotcha gotcha. sounds good, thanks!

Reply:
@AlicanAkyuz this change needs to be added to the design system before it can be added to Palette, so maybe we can create a ticket for design?

Reply:
@AlicanAkyuz i can do it :)